packages/meditrak-app-server/src/index.ts

```diff
@@ -10,7 +10,6 @@ import http from 'http';
 import winston from 'winston';
```

Comment:
(Unrelated to this file)
In package.json we have a script that calls `node dist/sync/createMeditrakSyncView.js`. The dist files for this may not be present. Can we use babel-node like we do in other packages for consistency? This also seems like a script so it should be outside the src dir.

Reply:
Done in: https://github.com/beyondessential/tupaia/pull/5063